pylama.ini

```diff
@@ -0,0 +1,6 @@
+[pylama]
+linters = pyflakes, pylint, mypy, pycodestyle
+skip = env3/*,.venv/*
+
+[pylama:pylint]
+ignore = C0111,W0611,C0301
```

Comment:
Please remove W0611 from ignore list. 
Also C0301 does not seem good idea - you may be able to solve it with this: https://github.com/klen/pylama/issues/17#issuecomment-256731547
100 chars seems like a reasonable limit.